updating/understanding_updates/intro-to-updates.adoc

```diff
@@ -0,0 +1,44 @@
+:_content-type: ASSEMBLY
+[id="understanding-openshift-updates"]
+= Introduction to OpenShift updates
+include::_attributes/common-attributes.adoc[]
+:context: understanding-openshift-updates
+
+toc::[]
+
+With {product-title} 4, you can update an {product-title} cluster with a single operation by using the web console or the OpenShift CLI (`oc`). Platform administrators can view new update options either by going to *Administration* -> *Cluster Settings* in the web console or by looking at the output of the `oc adm upgrade` command.
```

Comment:
I took out the mention of admins being "automatically notified when an update is available for their cluster."